editor/editor_node.cpp

```diff
@@ -378,7 +378,15 @@ void EditorNode::_fs_changed() {
 				// ensures export_project does not loop infinitely, because notifications may
 				// come during the export
 				export_defer.preset = "";
-				platform->export_project(preset, export_defer.debug, export_defer.path, /*p_flags*/ 0);
+				if (!preset->is_runnable() && (export_defer.path.ends_with(".pck") || export_defer.path.ends_with(".zip"))) {
```

Comment:
Do you really need to check for `is_runnable()`? AIUI the `runnable` is just the default preset for a given platform.

Reply:
I guess `is_runnable` not necessary, I have add it to allow additional control to export project as `zip` file